.github/workflows/publish-helmcharts-to-artifactory.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: "Push Helm Charts to Artifactory"
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: [master, v1.*]
```

Comment:
FMU: why also `v1.*` ?

Reply:
When running on v1.*, it uses v1.* as tag instead of the job number.
This would be the equivalent of releasing the new version, running on master would be just internal versions in between releases.

Reply:
Cool. Can you add an inline comment here indicating as much, for casual readers like me 🙂